src/resources/app/components/Layout/CreationLayout.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <v-app>
+    <v-container>
+      <div class="mt-16">
+        <v-btn
+          id="back"
+          color="white"
+          elevation="0"
+          style="padding-left: 0; padding-right: 8px"
+          @click="redirectBackToClasswork()">
```

Comment:
```suggestion
          @click="redirectBackToClasswork()"
        >
```
To make the hierarchy clearer